geode-redis/src/main/java/org/apache/geode/redis/internal/data/ByteArrayWrapper.java

```diff
@@ -39,6 +45,8 @@
    */
   protected byte[] value;
 
+  private transient Object routingId;
```

Comment:
Do we actually call the PartitionResolver more than once on a value? I wonder if we ought to just compute the routingID in the partition resolver and not bother caching the result. This is taking up extra memory.

Reply:
I would hope we only call it once for each incoming key operation. So yes, calculating in the resolver would work but then we leak the internals of `ByteArrayWrapper` to another class right? What if `ByteArrayWrapper.hashCode()` was the CRC16 value, then we would be good.

Reply:
Seems like it is called very often - certainly more than once / key